Extensions/anti_capitalism.js

```diff
@@ -1,5 +1,5 @@
 //* TITLE Anti-Capitalism **//
-//* VERSION 1.5.6 **//
+//* VERSION 1.5.7 **//
 //* DESCRIPTION	Removes sponsored posts, vendor buttons, and other nonsense that wants your money. **//
```

Comment:
this isn't related to the proposed changes but it would be nice to fix this erroneous tab here while we're working on this file

```suggestion
//* DESCRIPTION Removes sponsored posts, vendor buttons, and other nonsense that wants your money. **//
```